### DISCUSSION TITLE
GitHub Copilot - VS Code/VS2022 - Activation failed

### DISCUSSION THREAD

**Discussion by abug** [community/community#47589] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hello.

My GitHub Copilot installation, both for VS Code and VS 2022, has stopped working.

I've tried to reinstall GitHub Copilot extension, but the issue hasn't gone.

How else could this issue be solved?

VS Code output: 

[INFO] [auth] [2023-02-15T20:02:10.593Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-02-15T20:02:10.595Z] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support"

VS 2022 output:

GitHub Copilot 1.73.2.0 (v1.73.2.0@4c9585ed8)
GitHub Copilot Agent v1.73.0
You don't have access to GitHub Copilot.

**Comment by artemudovyk** [comment]
Related to: https://github.com/orgs/community/discussions/47577

**Comment by tuves** [comment, marked as answer]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

**Comment by PanjiDenisGiantoro** [comment]
github not update payment
